projects/angular/src/progress/progress-bars/progress-bar.ts

```diff
@@ -95,34 +95,41 @@ export class ClrProgressBar {
     return this._loop;
   }
 
-  /** @deprecated since 2.0, remove in 4.0 */
-  @Input('clrWarning')
-  set clrWarning(value: boolean | string) {
-    this._warning = isBooleanAttributeSet(value);
+  @Input('clrColor')
+  set clrColor(value: string) {
+    switch (value) {
+      case 'success':
+        this._success = true;
+        this._warning = false;
+        this._danger = false;
+        break;
+      case 'warning':
+        this._success = false;
+        this._warning = true;
+        this._danger = false;
+        break;
+      case 'danger':
+        this._success = false;
+        this._warning = false;
+        this._danger = true;
+        break;
+      default:
+        this._success = false;
+        this._warning = false;
+        this._danger = false;
+    }
```

Comment:
This change be simplified. See 9b9279d6ec0812b4bf39b0de47379a4d1c7da4c4.

Reply:
Thank you Kevin, looks much better.
Change implemented in commit  b6cb6bb

Reply:
Thanks!